src/network/network_models/powermodels_interface.jl

```diff
@@ -298,8 +298,8 @@ function PMvarmap(system_formulation::Type{S}) where {S <: PM.DCPlosslessForm}
     pm_var_map = Dict{Type,Dict{Symbol, Union{Symbol,NamedTuple}}}()
```

Comment:
Dispatch on the concrete like this `system_formulation::Type{PM.DCPlosslessForm}`

Reply:
this isn't a concrete type
```
julia> TypeTree(PowerSimulations.PowerModels.DCPlosslessForm)
PowerModels.DCPlosslessForm
├──StandardPTDFForm
└──PowerModels.NFAForm
```

Reply:
got it